data/events.txt

```diff
@@ -2054,46 +1345,15 @@ event "dreadnought deployment"
 		cargo 1
 		personality
 			heroic
-		variant 5
-			"Bastion"
-		variant 4
-			"Osprey"
-			"Fury"
-		variant 2
-			"Argosy"
-		variant 3
-			"Bastion"
-			"Osprey"
-		variant 2
-			"Bastion"
-			"Sparrow" 2
-		variant 1
-			"Falcon"
-		variant 1
-			"Falcon"
-			"Hawk" 2
-			"Sparrow" 3
-		variant 1
-			"Falcon"
-			"Bastion"
-			"Hawk"
-		variant 3
-			"Falcon (Plasma)"
-		variant 2
+		add variant 4
 			"Dreadnought"
-		variant 1
+		add variant 2
```

Comment:
Compering the original fleet definitions and the ones I replaced it looked like at some point the frequencies had been doubled. Where I have changed the frequencies the fleet definitions where very out of date.